src/server/license/email-address/controller.test.js

```diff
@@ -0,0 +1,148 @@
+import { createServer } from '~/src/server/index.js'
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+import { pageTitle } from './controller.js'
+import { withCsrfProtection } from '~/src/server/common/test-helpers/csrf.js'
+import { parseDocument } from '~/src/server/common/test-helpers/dom.js'
+
+describe('licenseEmailAddress', () => {
```

Comment:
no tests checking its been added to state correctly/incorrectly

Reply:
Done!